services/autoscaling/src/simcore_service_autoscaling/modules/buffer_machines_pool_task.py

```diff
@@ -42,7 +42,6 @@ async def _startup() -> None:
 
 def on_app_shutdown(app: FastAPI) -> Callable[[], Awaitable[None]]:
     async def _stop() -> None:
-        await stop_periodic_task(app.state.autoscaler_task)
```

Comment:
scarry looking change that I dont fully get, but 100% if this is intended

Reply:
yes that was a leftover from copy/pasting. actually the autoscale_task is handled in services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_task.py
this module only cares about the buffers_pool_task